packages/next-on-pages/src/buildApplication/processVercelFunctions/build.ts

```diff
@@ -83,24 +83,33 @@ type RelativePathOpts = {
 	relativeTo?: string;
 };
 
-// Chunks can contain `require("node:*")`, this is not allowed and breaks at runtime
-// the following fixes this by updating the require to a standard esm import from "node:*"
-export const nodeBuiltInModulesPlugin: Plugin = {
-	name: 'node:built-in:modules',
+/**
+ * Chunks can contain dynamic require statements for built-in modules. This is not allowed and
+ * breaks at runtime. The following fixes this by updating the dynamic require to a standard esm
+ * import from the built-in module.
+ *
+ * This applies to `require("node:*")` and `require("cloudflare:*")`.
+ */
+export const builtInModulesPlugin: Plugin = {
+	name: 'built-in:modules',
 	setup(build) {
-		build.onResolve({ filter: /^node:/ }, ({ kind, path }) => {
-			// this plugin converts `require("node:*")` calls, those are the only ones that
-			// need updating (esm imports to "node:*" are totally valid), so here we tag with the
-			// node-buffer namespace only imports that are require calls
+		build.onResolve({ filter: /^(node|cloudflare):/ }, ({ kind, path }) => {
+			/**
+			 * This plugin converts `require("prefix:*")` calls, those are the only ones that need
+			 * updating (esm imports to "prefix:*" are totally valid), so here we tag with the
+			 * built-in-modules namespace only imports that are require calls.
```

Comment:
Nit: instead of `prefix` I'd try to use something that makes it more apparent that it's a variable value

```suggestion
			 * This plugin converts `require("<PREFIX>:*")` calls, those are the only ones that need
			 * updating (esm imports to "<PREFIX>:*" are totally valid), so here we tag with the
			 * built-in-modules namespace only imports that are require calls.
```